gradle.properties

```diff
@@ -1,7 +1,7 @@
 projectVersion=1.4.5.BUILD-SNAPSHOT
```

Comment:
Can you bump the version to `2.0.0.BUILD-SNAPSHOT` since this changes the minimum version required to be Micronaut 1.3.4

Reply:
Oh, it works with 1.1.4. I just tested it with 1.3.4 and forgot to put it back to its initial value.